src/elfpy/utils/price.py

```diff
@@ -174,7 +235,7 @@ def calc_apr_from_spot_price(price: float, time_remaining: StretchedTime):
         "utils.price.calc_apr_from_spot_price: ERROR: "
         f"time_remaining.normalized_time should be greater than zero, not {time_remaining.normalized_time}"
     )
-    return (1 - price) / (price * time_remaining.normalized_time)  # price = 1 / (1 + r * t)
+    return (1 - price) / (price * time_remaining.normalized_time)  # r = ((1/s)-1)/t = (1-s)/(st)
```

Comment:
Nit: Would you mind using `p` instead of `s`? We use `p` pretty universally in our comments and docs when referring to spot price.